content/install_maintain/installation/manager-image.md

```diff
@@ -111,7 +111,7 @@ docker exec cfy_manager_local cfy_manager reset-admin-password NEW_PASSWORD
 After {{< param cfy_manager_name >}} is installed, you can configure your {{< param cfy_manager_name >}} for your environment, including:
 
 * [Upload plugins]({{< relref "working_with/official_plugins/_index.md" >}}) to add functionality to {{< param cfy_manager_name >}}
-* If you intend to use {{< param product_name >}} to work with LDAP, setup the [LDAP connection]({{< relref "working_with/console/pages/tenant-management-page.md" >}}).
+* If you intend to use {{< param product_name >}} to work with LDAP, setup the [LDAP connection]({{< relref "working_with/console/pages/tenants-page.md" >}}).
```

Comment:
I think something is wrong here. On Tenants Management page (now Tenants page) we don't have a way to setup LDAP connection. Maybe we should redirect user to https://docs.cloudify.co/staging/dev/working_with/manager/ldap-integration/ page?

Reply:
Done